src/pages/background-colors/BackgroundColors.tsx

```diff
@@ -14,11 +14,27 @@ const CustomContainer = styled.div`
 
 const StackContainer = styledMUI(Box)(({ theme }) => ({
   cursor: 'pointer',
+  position: 'relative',
   '&:hover': {
-    border: `5px solid ${theme.palette.success.main}`,
+    outline: `5px solid ${theme.palette.success.main}`,
+    h1: {
+      opacity: 1,
+    },
   },
 }));
 
+const Color = styled.h1<{ color: string }>`
```

Comment:
`color` is seem to be have only 1 value, but can I get your opinion about making it into variable?

Reply:
I thought that color object just have one value, then I write it like that
but be not convinced